bvt/run-bvt-setup

```diff
@@ -206,7 +241,11 @@ printf OUTF $cfgfile_fmt,
         $passwordipmi,
         $ffdcdir,
         $imagedir,
-        $imagename;
+        $imagename,
+	$lparip,
```

Comment:
Fixed the indentation issue in bvt/run-bvt-setup file by making the appropriate changes as required